docs/docs/coming-from-react-to-gatsby

```diff
@@ -0,0 +1,9 @@
+---
+title: Coming from React to Gatsby: What You Need to Know
+---
+
+This is a stub. Help our community expand it.
+
+The [Gatsby Style Guide](/docs/gatsby-style-guide) will help ensure your pull request gets accepted.
```

Comment:
The verb is missing kinda. Perhaps "Please review the Gatsby Style Guide to help ensure your pull request is accepted"?